### PR TITLE
Add CI runs every six hours and clean up naming conventions.

### PR DESCRIPTION
**Pull request by markusthoemmes**
**Note:** Most files are generated, only those in `config` should be reviewed.

1. Added CI runs every 6 hours to our latest 0.9 release.
2. Cleaned up naming conventions. Multi-version build configs now clearly state so in the file's suffix.
3. Removed the disfunctional vsphere run for now.

### CODE REVIEW

**Comment by mgencur** [comment]
/ltgm

Do you have email notification setup? I remember last time they were not implemented. I'm just curious how you will watch builds every six hours

File: ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-presubmits.yaml [hunk 1]
```diff
@@ -4,17 +4,17 @@ presubmits:
     always_run: true
     branches:
     - release-next
-    context: ci/prow/4.3-e2e-aws-ocp-43
+    context: ci/prow/4.2-e2e-aws-ocp-42
```
Comment (bbrowning) [inline ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-presubmits.yaml]: Are all the 4.3 -> 4.2 changes in the generated jobs expected? It looks like a corresponding 4.2 -> 4.3 change for each as well, perhaps? So just ordering issues in the diff?

Comment (markusthoemmes) [reply ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-presubmits.yaml]: yeah seems like ordering. I ditched the `__variant` and "unnamed" files and renamed them to be `__4.x` respectively.

**Pull request merged**